modules/ephemeral-storage-creating-configmaps-across-namespaces.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * work_with_shared_resources/using-shared-resource-csi-driver.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="ephemeral-storage-creating-configmaps-across-namespaces_{context}"]
+= Creating a SharedConfigMap instance
+
+You can create a `SharedConfigMap` CR instance for the `ConfigMap` object that you want to share across namespaces. 
+
+.Prerequisites
+
+You must have permissions to perform the following actions:
+
+* Create `sharedconfigmaps.sharedresource.openshift.io` custom resource definition (CRD) at a cluster-scoped level.
+* Create a `ClusterRole` object for the `SharedConfigMap` CR.
+* Create role and role binding for the Shared Resource Container Storage Interface (CSI) Driver.
+* Manage roles and role bindings across the namespaces in the cluster to control which users can get, list, and watch instances.
+* Manage roles and role bindings to allow the service account specified by a pod to use the `SharedConfigMap` CR to mount your `csi` volume that references a `ConfigMap` object .
+* Access the namespaces that contain the `ConfigMap` object you want to share.
+
+.Procedure
+
+. Create a `Role` and `RoleBinding` object to grant the CSI driver permission to access the `ConfigMap` object. See the following example configuration:
++
+.Example `Role` object
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: shared-test-config
+  namespace: <name_of_the_source_namespace> <1>
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    resourceNames: ["shared-config"]
+    verbs: ["get", "list", "watch"]
+----
+<1> Replace <name_of_the_source_namespace> with the name of the source namespace.
+
++
+.Example `RoleBinding` object
+[source,yaml]
+----
+ apiVersion: rbac.authorization.k8s.io/v1
+ kind: RoleBinding
+ metadata:
+   name: shared-test-config
+   namespace: <name_of_the_source_namespace> <1>
+ roleRef:
+   apiGroup: rbac.authorization.k8s.io
+   kind: Role
+   name: shared-test-config
+ subjects: <2>
+   - kind: ServiceAccount
+     name: csi-driver-shared-resource
+     namespace: openshift-builds
```

Comment:
Similarly, change <1> and <2> to # <1> and # <2> wherever applicable throughout the content. I will just add a global comment if I see this feedback later in the content.